runtime/layers/console/bootstrap

```diff
@@ -44,8 +44,9 @@ while (true) {
             $cliOptions = '';
         }
 
+        $timeout = $context->getRemainingTimeInMillis() / 1000 - 1;
```

Comment:
We may want to ensure this is always > 0, incase the timeout is 1 second, this will end up setting the timeout to zero.
```suggestion
        // Ensure the timeout is at least 1 second
        $timeout = max(1, $context->getRemainingTimeInMillis() / 1000 - 1);
```

Reply:
Good point, or even worse, the timeout passed to Symfony would be negative and it looks like the component throws in that case. Good catch 👍🏻